src/templates/content.html

```diff
@@ -25,14 +25,14 @@
     {% endif %}
     {% endwith %}
 
-    <h1>Video Index</h1>
+    <h1>Manage Uploads</h1>
```

Comment:
Manage uploads as a title doesn't make much sense to me as this page will show videos uploaded from all users.

Reply:
I think that it makes it look more like the renders. I agree that its an index of all videos and not just ones uploaded from the current user.

Reply:
I'll change it back.